frontend-react/src/components/FileHandlers/FileHandler.tsx

```diff
@@ -109,8 +109,8 @@ const FileHandler = ({
     }, [localError]);
 
     const { activeMembership } = useSessionContext();
-    const { organization, loading: organizationLoading } =
-        useOrganizationResource();
+    const { data: organization, isLoading: organizationLoading } =
```

Comment:
Do. you mind adding a comment in here about moving off of `isLoading` and onto using the `Suspense` component where possible? Our React Query client is set up to use Suspense and ErrorBoundaries for loading and catching.

Reply:
Made a comment. Let me know if the message is proper.

Reply:
perfect! thank you :)